### DISCUSSION TITLE
Date field when used in a Table field has no option to set the Time

### DISCUSSION THREAD

**Discussion by thisisjamessmith** [craftcms/cms#13395] [Ideas] [answered]
Much like the standard Date field shown in the screenshot, I feel like I should be allowed to optionally include the Time portion when used in a Table field, which is not currently possible. Can work around it with a separate "Time" column, but not ideal.

<img width="170" alt="image" src="https://github.com/craftcms/cms/assets/2416106/58cfd343-e7f5-4194-9d47-22d6083c525b">
<img width="738" alt="image" src="https://github.com/craftcms/cms/assets/2416106/436c0fe9-e64b-4517-a0c0-7d221940e82c">

**Comment by brandonkelly** [comment, marked as answer]
Each table cell is comprised of a single input, but there’s no single input for holding a date and time value (ignoring `<input type="datetime-local">`, which is terrible).
